File: hang.liu/950.js
var deckRevealedIncreasing = function(deck) {
  var sortedDeck = deck.sort((a, b) => a - b);
  var orderedDeck = [sortedDeck.pop()];
  while (sortedDeck.length > 0) {
    orderedDeck.unshift(sortedDeck.pop(), orderedDeck.pop());
  }
  return orderedDeck;
};

File: xiang.li/950.kt
import java.util.*

class Solution {
    fun deckRevealedIncreasing(deck: IntArray): IntArray {
        deck.sort()
        val queue: Queue<Int> = LinkedList()
        if (deck != null) {
            for (i in deck.indices.reversed()) { 
                queue.add(deck[i]) 
                if (i == 0) {
                    break
                }
                queue.add(queue.poll()) 
            }
        }
        if (deck != null) {
            for (i in deck.indices.reversed()) {
                deck[i] = queue.poll()
            }
        }
        return deck
    }
}
